errors/index.js

```diff
@@ -5,14 +5,13 @@ exports.handleCustomErrors = (err, req, res, next) => {
     else (next(err));
 };
 exports.handlePsqlErrors = (err, req, res, next) => {
-    if (err.code === '22P02') {
+    if (err.code = '22P02') {
```

Comment:
I would write the message as something like 'bad request' as this err code can be triggered by things other than an invalid id i.e. a wrong query